podvm/files/etc/systemd/system/kata-agent.service

```diff
@@ -6,9 +6,9 @@ After=netns@podns.service process-user-data.service
 
 [Service]
 ExecStartPre=mkdir -p /run/kata-containers
-ExecStart=/usr/local/bin/kata-agent --config /run/peerpod/agent-config.toml
+ExecStart=/usr/local/bin/kata-agent --config /etc/agent-config.toml
```

Comment:
I think the process-user-data will still update the default file - /run/peerpod/agent-config.toml (https://github.com/confidential-containers/cloud-api-adaptor/blob/main/cmd/process-user-data/types.go#L8).

Does the systemd unit also need to create a link 
`ln -s  /etc/agent-config.toml /run/peerpod/agent-config.toml`  ?

Reply:
oh, good catch. yes, we also need to undo that, otherwise cc_kbc will also be broken on packer-based builds

Reply:
Ok, with the path reset in `types.go` cc_kbc works again on a packer-built image. It doesn't work w/ an mkosi-image, but that's expected.